migrate/index.md

```diff
@@ -18,35 +18,50 @@ which are:
 - How much data you are migrating (megabytes, or terabytes?)
 - Where you are migrating your data from (PostgreSQL, TimescaleDB, Influx, or MySQL?)
 
+Below are the different migration options we offer. You can choose the one that best suits your requirements:
+
+| Migration strategy | Supported sources | Use when | Downtime requirements |
+| ---------------- | --- | --- | --- |
+| [pg_dump and pg_restore][pg-dump-restore] | Postgres,  TimescaleDB | Downtime is okay | Requires some downtime |
+| [Dual writes and backfill][dual-write] | Postgres, TimescaleDB and others | Append-only data, heavy insert workload (~20,000 inserts per second) | Optimized for minimal downtime |
+| [Live migrations][live-migrations] | Postgres | Simplified end-to-end migration with almost zero downtime | Optimized for minimal downtime |
+
 If you are using PostgreSQL or TimescaleDB and can afford to take your
 application offline for a few hours, the simplest option is to migrate data
 from another database into Timescale using PostgreSQL's `pg_dump` and
-`pg_restore` commands.
-You can also use these tools to migrate your data from Managed Service for
-TimescaleDB, from a self-hosted TimescaleDB instance, or from another
-PostgreSQL database, including Amazon RDS. Consult the guide on [migrating with
-pg_dump and pg_restore][pg-dump-restore].
-
-If you are looking for a low-downtime alternative (downtime on the order of
-minutes), are willing to modify your ingestion code, and the bulk of your data
-is stored in time-series tables, you can use the [dual-write and backfill][dual-write]
-strategy for a low-downtime migration. This strategy also works if you are not
-migrating from PostgreSQL.
+`pg_restore` commands. 
+
+You can also use these tools to migrate your data from Managed Service for TimescaleDB, from a self-hosted TimescaleDB instance, or from another
```

Comment:
We don't support Timescale as a source yet.

Reply:
Yes, thats the reason I have added table in the top mentioning the supported sources. In this sentence the context is we have tools that support migrations from Postgres, Timescale and others.